chimedb/data_index/__init__.py

```diff
@@ -17,11 +17,13 @@
     HKAcqInfo,
     HKFileInfo,
     HKPFileInfo,
+    MiscFileInfo,
+    MiscFileList,
     RawadcAcqInfo,
     RawadcFileInfo,
     StorageGroup,
     StorageNode,
     WeatherFileInfo,
 )
 
-__version__ = "0.0.2"
```

Comment:
I've moved to versioneer and calendar versioning on pretty much all other packages. Should we consider doing that here?

Reply:
That sounds like a different PR...